app/common/util/src/main/java/io/syndesis/common/util/Labels.java

```diff
@@ -15,17 +15,16 @@
  */
 package io.syndesis.common.util;
 
-import java.util.Locale;
 
 public final class Labels {
 
-    private static final String VALID_REGEX = "(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?";
+    private static final String VALID_REGEX = "(([A-Za-z0-9][-A-Za-z0-9_]*)?[A-Za-z0-9])?";
```

Comment:
I think . is ok according to https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set:

> Valid label values must be 63 characters or less and must be empty or begin and end with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between

'would call it `VALID_VALUE_REGEX` to not confuse it with the label name.

Reply:
minor nit: would use non-capturing grouping to be clear that is about grouping, not capturing (not important though)